XLS-30/README.md

```diff
@@ -0,0 +1,1254 @@
+
+  - Our Innovative Solution: To achieve the above mentioned, we introduce a mechanism for the AMM instance to continuously auction-off trading advantages for a 24-hour slot at zero trading fee! Anyone can bid for the auction slot with the units of **`LPtokens`**. The slot-holder can send the arbitrage transaction **immediately** without the need to wait for their profits to exceed the trading fee, thus eliminating the race condition for them. This also reduces the **time window** for which the pool suffers decreased trading volume. Additionally, part of proceeds (**`LPTokens`**) from the auction are deleted/burnt that effectively increases LP token holders' ownership in the pool proportionally. Since it's a continuous auction mechanism, if someone outbids an auction slot-holder, part of proceeds from the auction are refunded to the previous slot-holder (computed pro-rata). For details refer Section 5.
+
+As slot holder will have significant advantages for arbitrage, it’s expected that arbitrageurs will bid up the price of the auction slot to nearly the value they extract through arbitrage.
+
+  - Expected Results
+     - Eliminates wait time & race condition for auction slot holder (arbitrageur)
+     - Narrows time windows in which the pool suffers decreased trading volume
+     - Liquidity providers additionally reap a share of profits that would otherwise go JUST to arbitrageurs
+
+## 2. Creating AMM instance on XRPL
+
+### 2.1. On-Ledger Objects
+
+We propose the existing **`AccountRoot`** object together with a new ledger object **`AMM`** to represent an AMM instance. Currently, the **`AccountRoot`** object type describes a single account, its settings, and XRP balance on the XRP ledger. The **`AccountRoot`** and the **`AMM`** objects that represent an AMM instance can be created using the new **`AMMCreate`** transaction. XRP balance of the AMM instance pools is always tracked via the existing `Balance` field of the **`AccountRoot`** object. The issued asset balances of the AMM instance pools and **`LPTokens`** are automatically tracked via trustlines. The AMM can be traded against using the new **`AMMDeposit`** and **`AMMWithdraw`** and the existing **`Payment`** transactions. Additionally, the AMM instance in "empty" state can be deleted using the `AMMDelete` transaction.
+
+Note that in this version, only equal-weighted two asset pools are supported. However, differing weighted pools could be supported in future versions.
+
+#### 2.1.1. Ledger Entries representing AMM instance
+
+##### **`AccountRoot`** ledger entry
+
+A new flag **`lsfAMM`** is introduced to the [**`AccountRoot`**](https://xrpl.org/accountroot.html) object. 
+
+| Flag Name |  Hex Value | Description |
+| ---------- | :----------------: | :-------:|
+| `lsfAMM`      | 0x40000000  |   If set, it indicates that the object represents an AMM instance|
```

Comment:
`lsfAMM` is obsolete and is replaced with `AMMID`.

Reply:
Deleted.